docker/Dockerfile.agent

```diff
@@ -1,17 +1,15 @@
-FROM python:3.10
-
-COPY README.md pyproject.toml poetry.lock /src/
-COPY beamer /src/beamer
+FROM python:3.10-alpine
 
+COPY . /src
 RUN mkdir -p /src/beamer/data/relayers
-COPY ./relayer/relayer-node16-*-x64 /src/beamer/data/relayers/
-
+COPY ./relayer/relayer-node18-linux-x64 /src/beamer/data/relayers/
```

Comment:
We should copy both relayers since we want to use the same image on both platforms.

Reply:
Nevermind the above comment. People on mac can use the same alpine-based image and it will work.